### PR TITLE
Stakeconomy Inflation Candidate Nomination

### PR DESCRIPTION
**Pull request by stakeconomy**
#### Problem
Submit Stakeconomy.com nomination
#### Summary of Changes
Added our keys

### CODE REVIEW

File: sdk/src/feature_set.rs [hunk 1]
```diff
@@ -49,6 +49,15 @@ pub mod full_inflation {
         }
     }
 
+    pub mod stakeconomy {
+        pub mod vote {
+             solana_sdk::declare_id!("JCergKv4GcywaBzn4JHi3sYJfG7mWenTG3QQDNUJiGS");
```
Comment (mvines) [inline sdk/src/feature_set.rs:54]: Hey @stakeconomy, the code beautifier is complaining here.  Can you please make this edit:
```suggestion
            solana_sdk::declare_id!("JCergKv4GcywaBzn4JHi3sYJfG7mWenTG3QQDNUJiGS");
```

Comment (mvines) [reply sdk/src/feature_set.rs]: Oh, looks like github let me do it.  So fixed :)

**Comment by mvines** [comment]
Thanks @stakeconomy, you're all set now!

**Comment by stakeconomy** [comment]
Great! Thanks Michael

Op 30 jan. 2021 om 19:06 heeft Michael Vines <notifications@github.com> het volgende geschreven:

﻿

Thanks @stakeconomy<https://github.com/stakeconomy>, you're all set now!

**Pull request merged**